modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/IdleVerifyResultV2.java

```diff
@@ -322,9 +275,9 @@ private void printConflicts(Consumer<String> printer) {
 
                 printer.accept("Partition instances: " + entry.getValue() + "\n");
             }
-
-            printer.accept("\n");
         }
+
+        printer.accept("\n");
```

Comment:
Lets keep output format as is (and for `counterConflicts()` too)

Reply:
Fixed.